README.md

```diff
@@ -69,3 +69,19 @@ example.com {
 
 This configuration would ignore any mDNS records that do not contain the
 string "my-id" in their service name.
+
+If `bind address` is specified in the configuration, the plugin will only send
+mDNS traffic to the associated interface. This prevents sending multicast
+packets on interfaces where that may not be desirable. To use `bind address`
+without setting a filter, set `filter text` to "".
+
+~~~ corefile
+example.com {
+    mdns example.com 3 "" 192.168.1.1
+}
+~~~
+
+This configuration will only send multicast packets to the interface assigned
+the `192.168.1.1` address. The interface lookup is dynamic each time an mDNS
```

Comment:
How expensive is the lookup before each query is sent?

Reply:
Not very. It takes 1 to 2 ms on my system, which is insignificant compared to the time it takes to actually do the query.

Also worth noting that this runs in a background thread, so it doesn't impact response time of the DNS server at all.